sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_models.py

```diff
@@ -91,7 +91,7 @@ def _from_generated(cls, generated):
         props.has_immutability_policy = generated.properties.has_immutability_policy
         props.has_legal_hold = generated.properties.has_legal_hold
         props.metadata = generated.metadata
-        props.encryption_scope = FileSystemEncryptionScope._from_generated(generated)  #pylint: disable=protected-access
+        props.encryption_scope = EncryptionScopeOptions._from_generated(generated)  #pylint: disable=protected-access
```

Comment:
Do we think this should use the `_options` suffix to match?
I don't feel to strongly on it - and it's not blocking for preview - though I'm curious what other languages did.